cmd/better-retester/main.go

```diff
@@ -29,10 +22,10 @@ type options struct {
 	config configflagutil.ConfigOptions
 	github prowflagutil.GitHubOptions
 
-	dryRun bool
+	runOnce bool
+	dryRun  bool
 
-	orgRaw prowflagutil.Strings
-	orgs   sets.String
```

Comment:
I ended removing this because I needed to remove the code that used it. There was a 

```
if !orgs.Has(org) {
  continue
}
```

guard clause in the `query` method which made the code skip all queries unless the organization was explicitly allowlisted by this parameter, and I did not understand its purpose.

Reply:
It was to mimik the current tester:

https://github.com/openshift/release/blob/1c62bf4564daeda670edae0f89730104583640ca/ci-operator/jobs/infra-periodics.yaml#L222-L223

I was thinking if we want to enable the retester for all repos or for no repos by default.